Tasks/AzureWebAppV1/taskparameters.ts

```diff
@@ -55,18 +56,20 @@ export class TaskParametersUtility {
     private static async getWebAppKind(taskParameters: TaskParameters): Promise<any> {
         var resourceGroupName = taskParameters.ResourceGroupName;
         var kind = taskParameters.WebAppKind;
-        if (!resourceGroupName) {
-            var appDetails = await AzureResourceFilterUtility.getAppDetails(taskParameters.azureEndpoint, taskParameters.WebAppName);
-            resourceGroupName = appDetails["resourceGroupName"];
-            if(!kind) {
-                kind = webAppKindMap.get(appDetails["kind"]) ? webAppKindMap.get(appDetails["kind"]) : appDetails["kind"];
+        if (taskParameters.azureEndpoint.scheme.toLowerCase() != AzureRmEndpointAuthenticationScheme.PublishProfile) {
+            if (!resourceGroupName) {
```

Comment:
so is resourceGroupName always expected to be available with publish profile scheme?